src/pages/Signin/Loading.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect } from 'react';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+import styled from 'styled-components';
+
+const REST_API_KEY = process.env.REACT_APP_REST_API_KEY;
+
+function Loading() {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const code = searchParams.get('code');
+  const bodyData = {
+    grant_type: 'authorization_code',
+    client_id: REST_API_KEY,
+    redirect_uri: 'http://localhost:3000/oauth/callback/kakao',
+    code: code,
+  };
+  const queryStringBody = Object.keys(bodyData)
+    .map(k => encodeURIComponent(k) + '=' + encodeURI(bodyData[k]))
```

Comment:
```suggestion
    .map(k => encodeURIComponent(k) + '=' + encodeURI(bodyData[k]))
```
매개변수 k가 의미하는 데이터가 어떤데이터 일지 유추할 수 있게 적어주세요.
내가 올린 코드는 나만 보는게 아니라, 팀원들도 보는 코드입니다.
다른사람도 쉽게 예상할 수있게 이름짓는게 정말 중요해요.

Reply:
수정이 안됐습니다